LICENSES/ISC.txt

```diff
@@ -0,0 +1,14 @@
+ISC License Copyright (c) 2004-2010 by Internet Systems Consortium, Inc. ("ISC")
+
+Copyright (c) 1995-2003 by Internet Software Consortium
+
+Permission to use, copy, modify, and /or distribute this software for any
+purpose with or without fee is hereby granted, provided that the above copyright
+notice and this permission notice appear in all copies.
+
+THE SOFTWARE IS PROVIDED "AS IS" AND ISC DISCLAIMS ALL WARRANTIES WITH REGARD
```

Comment:
In the few versions I looked at[1], it says "THE AUTHOR" instead of ISC, except on spdx.org where those parts are in red, which I interpret as "it needs to be changed".
What do you think?

[1]:
- https://opensource.org/licenses/ISC
- https://directory.fsf.org/wiki/License:ISC
- https://git.kernel.org/pub/scm/linux/kernel/git/jberg/iw.git/tree/COPYING